src/Controls/src/Core/Platform/Android/BottomNavigationViewUtils.cs

```diff
@@ -88,12 +118,17 @@ internal static void UpdateEnabled(bool tabEnabled, IMenuItem menuItem)
 
 			if (loadTasks.Count > 0)
 				await Task.WhenAll(loadTasks);
+		}
 
-			foreach (var menuItem in menuItems)
-				menuItem.Dispose();
+		internal static void SetMenuItemTitle(IMenuItem menuItem, string title)
+		{
+			using (var jTitle = new Java.Lang.String(title))
```

Comment:
```suggestion
			using var jTitle = new Java.Lang.String(title);
```
And remove not needed indentation